stackle_api/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:boron
+
+#create the app directory
+WORKDIR /usr/src/app
+
+COPY package.json .
+
+RUN npm install
+
+COPY . .
```

Comment:
There is no particular reason. Just went with the flow i thought was good since i didn't come across any guidelines as to which should be done when. Is there are particular order to be followed?

Reply:
Each of these  RUN, COPY, etc commands adds a layer to your final Docker image, try always 
 to avoid unnecessary layers. `COPY . .` followed by `RUN npm install` would do.